libs/scripts/scripts.js

```diff
@@ -123,7 +123,8 @@ const config = {
   geoRouting: 'on',
   fallbackRouting: 'on',
   links: 'on',
-  imsClientId: 'milo',
+  // will be reverted before merging
+  imsClientId: 'fedsmilo',
```

Comment:
This, of course, will need to be removed before adding the 'Ready for Stage' label